src/features/lightspeed/api.ts

```diff
@@ -76,6 +76,9 @@ export class LightSpeedAPI {
       console.error("Ansible Lightspeed instance is not initialized.");
       return {} as CompletionResponseParams;
     }
+    console.log(
```

Comment:
Can we add a prefix for these logs, like we did for the previous features? Something like: `[ansible-lightspeed]`.

Reply:
Fixed